discovery/discovery.go

```diff
@@ -148,12 +148,12 @@ func doDiscover(app App, pre string, insecure bool) (*Endpoints, error) {
 // DiscoverEndpoints will make HTTPS requests to find the ac-discovery meta
```

Comment:
I am thinking that we would really just expose one external function, `DiscoverEndpoints`, which would encapsulate both simple and meta.